core/src/main/java/org/sonarsource/sonarlint/core/container/connected/update/perform/ServerIssueUpdater.java

```diff
@@ -49,7 +49,7 @@ public void update(String projectKey, Sonarlint.ProjectConfiguration projectConf
   }
 
   public void updateServerIssues(String projectKey, Sonarlint.ProjectConfiguration projectConfiguration, Path path, boolean fetchTaintVulnerabilities, ProgressWrapper progress) {
-    List<Sonarlint.ServerIssue> issues = issueDownloader.download(projectKey, projectConfiguration, fetchTaintVulnerabilities, progress);
+    List<Sonarlint.ServerIssue> issues = issueDownloader.download(projectKey, projectConfiguration, fetchTaintVulnerabilities, null, progress);
```

Comment:
Note: If I understand correctly, this is where we should inject the computed remote branch?

Reply:
Yes. And also we have a call from PartialUpdater. I'm not sure about it.